apps/meteor/deploy/README.md

```diff
@@ -0,0 +1,146 @@
+- [Deploying](#deploying)
+  - [Files](#files)
+  - [Environment variables](#environment-variables)
+  - [Basic configuration](#basic-configuration)
+  - [Deploying without MongoDB authentication](#deploying-without-mongodb-authentication)
+  - [Deploying with MongoDB authentication](#deploying-with-mongodb-authentication)
+  - [Deploying with HTTPS](#deploying-with-https)
+  - [Deploying with HTTPS (including MongoDB authentication)](#deploying-with-https-including-mongodb-authentication)
+  - [Migrating from exiting the compose file](#migrating-from-exiting-the-compose-file)
+
+# Deploying
+
+*These compose files are compatible only with compose v2. You can install compose v2 here:- https://docs.docker.com/compose/cli-command/*
+*Compose v1 is no longer supported.*
+
+## Files
+
+There are two files, 
+* `compose.yml` this is similar to your old compose file.
+* `compose-auth.yml` this template makes enabling mongodb authentication easier.
+* `compose-https.yml` this template includes traefik to help with deployments with TLS cert and FQDN.
+* `compose-auth-https.yml` same as `compose-auth.yml` with easy HTTPS setup.
+
+## Environment variables 
+
+| RELEASE         | Rocket.Chat Release (version); defaults to the latest (:latest tag)  |
+|-----------------|----------------------------------------------------------------------|
+| MONGODB_VERSION | MongoDB Version (defaults to 4.4)                                    |
```

Comment:
I'm curious to understand why going with version 4.4? Version 5.0 is [supported](https://docs.rocket.chat/getting-support#mongodb-support), [tested](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/workflows/build_and_test.yml#L179) and likely to be supported for a longer period.
If version 4.4 is "recommended" over 5.0 I would suggest to add such information in the [documentation](https://docs.rocket.chat/getting-support#mongodb-support).

Reply:
This PR was started 2021 so I'm guessing it was that 🙈

Reply:
actually the reason is that 5.0 does not work some specific architectures. read more [here](https://www.mongodb.com/docs/manual/administration/production-notes/?_ga=2.126650885.920302513.1651862981-45560890.1651300147#x86_64). I'd rather have people be able to start and use rocketchat and slowly figure out how to update to 5.0 than facing a blocker right from the beginning with an unambiguous error message 🙂 Remember, 4.4 doesn't reach EoL until feb 2024.

Reply:
Thanks a lot for the explanation, totally make sense 👍